erlang/atomvm/Portfile

```diff
@@ -1,8 +1,13 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
-PortGroup               github 1.0
 PortGroup               cmake 1.1
+PortGroup               compiler_blacklist_versions 1.0
+PortGroup               legacysupport 1.1
+PortGroup               github 1.0
```

Comment:
if you change the order to alphabetize, please at least do so correctly 😉

Reply:
Okay, sure, will move this to a separate commit.

Reply:
what I meant is to use the correct alphabet - last time I checked "g" cones before "l".

Reply:
Indeed, I need more caffeine :) 
Will re-fix it.